nexus/types/src/external_api/shared.rs

```diff
@@ -90,16 +90,61 @@ pub enum IdentityType {
     SiloGroup,
 }
 
+/// Describes how identities are managed and users are authenticated in this
+/// Silo
+#[derive(Clone, Debug, Deserialize, Serialize, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub enum SiloIdentityMode {
+    /// Users are authenticated with SAML using an external authentication
+    /// provider.  The system updates information about users and groups only
+    /// during authentication (i.e,. "JIT provisioning" of users and groups).
```

Comment:
> The system updates information about users and groups only during authentication

Nit: only as a result of successful authentication.

Reply:
Fixed in 394c3be9.